packages/components/pie-switch/src/index.ts

```diff
@@ -94,13 +94,7 @@ export class PieSwitch extends FormControlMixin(RtlMixin(LitElement)) implements
     onChange (event: Event) {
         const { checked } = event?.currentTarget as HTMLInputElement;
         this.checked = checked;
-        const changedEvent = new CustomEvent(
-            ON_SWITCH_CHANGED_EVENT,
```

Comment:
There's a ESLint warning after this was removed: 'ON_SWITCH_CHANGED_EVENT' is defined but never used